test/types/findByPk.ts

```diff
@@ -1,3 +1,20 @@
+import { expectTypeOf } from 'expect-type'
+import { Attributes } from '@sequelize/core';
 import { User } from './models/User';
 
-User.findByPk(Buffer.from('asdf'));
+async () => {
+    const user = await User.findByPk(Buffer.from('asdf'));
+    expectTypeOf(user).toEqualTypeOf<User | null>()
+
+    const rawUser = await User.findByPk(123, { raw: true });
```

Comment:
can you add a test for `raw:true & rejectOnEmpty:true`, as well as `raw: false & rejectOnEmpty:true` to make sure they still work properly?